perceval/components/_pauli.py

```diff
@@ -102,3 +104,47 @@ def get_pauli_gate(pauli_type: PauliType):
         return Matrix([[1, 0], [0, -1]])
     else:
         raise NotImplementedError(f"{pauli_type}")
+
+
+def prep_state_circuit_preparer(prep_state_indices: List):
+    for i, pauli_type in enumerate(prep_state_indices):
+        yield i * 2, get_preparation_circuit(pauli_type)
+
+
+def meas_state_circuit_preparer(pauli_indices: List):
+    for i, pauli_type in enumerate(pauli_indices):
```

Comment:
"""
    Builds a measurement circuit to measure photons created in the Pauli Basis (I,X,Y,Z) to perform
    tomography experiments.
    :param pauli_indices: List of 'n'(=nqubit) indices to choose a circuit to measure the prepared state at nth qubit
    """

Reply:
oops, I will add them. Thanks for noticing it